hook!-full-game/src/Player/CameraRig.gd

```diff
@@ -6,6 +6,7 @@ onready var _camera: Camera2D = $Camera2D
 export var offset: = Vector2(200.0, 160.0)
 export var mouse_range: = Vector2(100.0, 500.0)
 
+var inactive := false
```

Comment:
Let's use `active` instead, it's the name used in Godot's nodes already for similar properties